go.mod

```diff
@@ -4,15 +4,21 @@ go 1.16
 
 require (
 	github.com/gorilla/mux v1.8.0
+	github.com/onsi/ginkgo v1.16.4 // indirect
 	github.com/onsi/ginkgo/v2 v2.1.1
 	github.com/onsi/gomega v1.18.1
+	github.com/openshift-online/ocm-sdk-go v0.1.241 // indirect
 	github.com/prometheus/alertmanager v0.23.0
 	github.com/prometheus/client_golang v1.11.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.3.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/kubectl v0.23.3
 	github.com/spf13/viper v1.10.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/cli-runtime v0.22.1 // indirect
+	k8s.io/client-go v0.22.1 // indirect
```

Comment:
We might need to pin the indirect deps under `replace` directive. Running `go mod tidy` for the clone of the PR is changing state of `go.mod` file for now.

Reply:
Ahh, thanks for catching that - no, that's just a goof-up on my part. Fixed now.